test/simulator/main/org/apache/cassandra/simulator/harry/HarryValidatingQuery.java

```diff
@@ -137,9 +138,8 @@ protected Object[][] executeNodeLocal(String statement, TokenPlacementModel.Node
                 catch (Throwable t)
                 {
                     logger.error("Caught an exception while validating", t);
```

Comment:
Should we turn into a suppressed error as above here, too?

Reply:
> suppressed error

why would we want to suppress?  Something unexpected happened and we have no clue how to handle... so we should fail the test.  This might be the `AssertError` we now generate, but we have to push that up... but what if its a different error?